### ISSUE TITLE
[4.0] Error in configuration of com_content

### ISSUE THREAD

**Issue by astridx** [joomla/joomla-cms#19543]
### Steps to reproduce the issue
Open configuration of com_content - see attached screen shot. (Global configuration | on the left Article | then tab List Layouts

### Expected result
You see only configuration options.

### Actual result
You see an error in the end of the configuration options.

![articles options test administration](https://user-images.githubusercontent.com/9974686/35752331-d46e6898-085b-11e8-8be5-d654089c98a7.png)

**Comment by anuragteapot** [comment]
Issue confirmed.

![screenshot from 2018-02-02 23-02-59](https://user-images.githubusercontent.com/30978328/35758754-6632b37c-086d-11e8-8f1c-5a09be2c6130.png)

**Comment by anuragteapot** [comment]
I thought that this one ia a issue but.
You have to enable Content <code>vote plugin</code> to see these options.

**Comment by ReLater** [comment]
If you don't enable it you shouldn't see these messages but just nothing or maybe a hint that plugin is disabled.

**Comment by anuragteapot** [comment]
@C-Lodder what for <code>type=voteradio</code> there should be no error if plugin is disable ?

**Comment by C-Lodder** [comment]
This is causing the issue: https://github.com/joomla/joomla-cms/blob/4.0-dev/layouts/joomla/form/field/radio.php#L75

Move this up to line **57**, so it becomes:

```
<?php if (strpos(trim($class), 'switcher') !== false && !empty($options)) : ?>
```

and it will work

**Comment by anuragteapot** [comment]
@C-Lodder it works. 
Can I make PR ?

**Comment by C-Lodder** [comment]
Yup, it works. Feel free to make a PR

**Comment by C-Lodder** [comment]
@Anu1601CS There's already a PR to split radio/switcher into 2 separate layouts. So I've asked Dimitris to do this in his current PR.

**Comment by anuragteapot** [comment]
Ok Done! let Dimitris add this fix to his current PR.

**Comment by anuragteapot** [comment]
@astridx Please test this PR #19296  and close this issue.

**Comment by ghost** [comment]
@Anu1601CS #19296 is tested successfully by @astridx 15 Days ago.

**Comment by anuragteapot** [comment]
@franz-wohlkoenig yeah , but this issue is also solved in #19296.
So, this issue should be close.

**Comment by ghost** [comment]
closed as stated above.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19543">issues.joomla.org/tracker/joomla-cms/19543</a>.</sub>
